packages/gatsby/cache-dir/normalize-page-path.ts

```diff
@@ -0,0 +1,25 @@
+export function normalizePagePath(path: string): string {
+  if (path === undefined) {
+    return path
+  }
+  if (path === `/`) {
+    return `/`
+  }
+  if (path.charAt(path.length - 1) === `/`) {
+    return path.slice(0, -1)
+  }
+  return path
+}
+
+export function denormalizePagePath(path: string): string {
```

Comment:
if this (`denormalizePagePath `) is not used in cache-dir - we should not add it here